forui/lib/src/widgets/text_field/text_field.dart

```diff
@@ -0,0 +1,691 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
```

Comment:
Do we required both `material.dart` and `cupertino.dart`?

Reply:
Yes we do, we currently default text selection & text selection control colors to `CupertinoColors.activeBlue`.